pyproject.toml

```diff
@@ -1,3 +1,9 @@
+[virtualenvs]
```

Comment:
This was necessary to get pyright via pre-commit to play nicely with poetry. We store the venv locally (kinda like having a local `node_modules`) and then pyright can reference it.